types/listlike_partial_deserializing_iterator.hh

```diff
@@ -84,6 +85,7 @@ private:
             : _remain(0) {
     }
 public:
+    listlike_partial_deserializing_iterator() = default;
```

Comment:
Is it OK that default-constructed iterator will be dangerous to use because of uninitialized _in pointer and _remain integer?

Reply:
Iterators are analogs of pointers, and a pointer default constructor also leaves it uninitialized.